### ISSUE TITLE
error: can't copy 'build/temp.linux-x86_64-cpython-311/symengine_install/lib/python3.11/site-packages/symengine/lib/symengine_wrapper.cpython-311-x86_64-linux-gnu.so': doesn't exist or not a regular file

### ISSUE THREAD

**Issue by KOLANICH** [symforce-org/symforce#330]
**Describe the bug**
The whole `build/temp.linux-x86_64-cpython-311/symengine_install/lib/python3.11` doesn't exist.

`find` shows that the file is are in 
`build/temp.linux-x86_64-cpython-311/symengine_install/local/lib/python3.11/dist-packages/symengine/lib/symengine_wrapper.cpython-311-x86_64-linux-gnu.so`

**To Reproduce**
Steps to reproduce the behavior, e.g.:
1. `python3 -m build -nwx .`

**Expected behavior**
Should be built.

**Environment (please complete the following information):**
 - OS and version: Debian sid
 - Python version 3.11
 - SymForce Version c3d2f3c97d77e8d9015276cd9ae7b080df646fc3

**Comment by realSaddy** [comment]
I am also facing this issue with python310. Did you have any luck getting it built? @KOLANICH

**Comment by aaron-skydio** [comment]
Try #342 ?

**Comment by KOLANICH** [comment]
>Did you have any luck getting it built?

Yes, I have built it. But I have done it semi-manually, somewhere editing `CMakeCache.txt`, somewhere replacing pieces of files with own code, somewhere hardcoding paths, somewhere using `patchelf` utility. It took me a lot of time to just build this "masterpiece of marvelous engineering" into a wheel that can be installed with `pip` and with which the examples from the `ReadMe` work. And I had to unload X11 and almost all the system services in order just to have enough RAM to build native code part of this package. The mere invocations of the compiler took a lot of time in my system. Since it is such problematic for me to rebuild the package, I'm not going to try #342 unless I really have to do the painful rebuilding process again.
